### DISCUSSION TITLE
Solving enviroments indefinitely

### DISCUSSION THREAD

**Discussion by kyleluma** [idaholab/moose#17981] [Q&A Getting Started] [answered]
I have just began installing Moose. I've done all of the steps correctly I am assuming since everything has worked out fine prior to this.
I run:
conda create --name moose moose-libmesh moose-tools

to which it succesfully collects package metadata, attempts to solve enviroment but fails:
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done

and then:
Solving enviroment: (with the spinning line)

I've quit and restarted from the installation of minconda and it says everything is where it should be, not sure what the problem would be.
Thanks

**Comment by milljm** [comment, marked as answer]
Strange issue more and more folks are experiencing from Anaconda cloud servers (https://github.com/conda/conda/issues/9554).
Can you perform the following, and then try to install packages?
```bash
conda config --set ssl_verify False
conda create --name moose -q -y
conda activate moose
conda install moose-tools
conda install moose-libmesh
```
Note about the additional steps; We are also finding Conda Solves to be much quicker when asking Conda to install individual packages. Start with moose-tools, as that package contains the most non-version specific amount of dependencies. Once complete (if --set ssl_verify worked), then install moose-libmesh.

**Reply by kyleluma** [reply]
thanks a lot for this!! God bless.

**Reply by milljm** [reply]
I forgot to mention: We are in the process of making the above instructions the norm (optimize install times): https://github.com/idaholab/moose/issues/17934
